dev-python/axolotl-curve25519/axolotl-curve25519-0.4.1_p2.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eapi7-ver
+
+MY_PN="python-${PN}"
+MY_PV="$(ver_cut 1-3)-$(ver_cut 5)"
+
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} )
+
+inherit distutils-r1 eapi7-ver
+
+DESCRIPTION="A python wrapper for the curve25519 library with ed25519 signatures"
+HOMEPAGE="https://github.com/tgalal/python-axolotl-curve25519"
+SRC_URI="https://github.com/tgalal/${MY_PN}/archive/${MY_PV}.tar.gz -> ${P}.tar.gz"
```

Comment:
Use MY_PN-MY_PV as distfile name to match directory from S.

Reply:
Done.